statistics/handle/internal/cache/lru/lru_cache.go

```diff
@@ -12,26 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package handle
+package lru
 
 import (
 	"container/list"
 	"math"
 	"sync"
 
 	"github.com/pingcap/tidb/statistics"
-	handle_metrics "github.com/pingcap/tidb/statistics/handle/metrics"
+	"github.com/pingcap/tidb/statistics/handle/internal/cache"
+	"github.com/pingcap/tidb/statistics/handle/metrics"
 )
 
-type statsInnerCache struct {
+// StatsInnerCache is the LRU cache for statistics.
+type StatsInnerCache struct {
 	elements map[int64]*lruMapElement
 	// lru maintains item lru cache
 	lru *innerItemLruCache
 	sync.RWMutex
 }
 
-func newStatsLruCache(c int64) *statsInnerCache {
-	s := &statsInnerCache{
+// NewStatsLruCache creates a new LRU cache for statistics.
+func NewStatsLruCache(c int64) *StatsInnerCache {
```

Comment:
Better to return the interface `StatsCacheInner` instead of the structure pointer `*StatsInnerCache`.
The detailed Implementation of the interface should be hidden.

Reply:
I think it's okay here, as long as the usage is through an interface.